index.html

```diff
@@ -147,6 +161,7 @@ <h1>I’m Abhishek <br>
             </div>
         </div>
     <div class="describe">
+        <div class="abt-self">
         <h3>About <br> Myself</h3>
         <p class="description">Hello I’m a software
```

Comment:
- [ ] Kindly remove the `line breaks (<br>)` you added to your about-me description so that the text can scale appropriately as you resize the page 👍🏽 .